eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -361,7 +454,7 @@ impl Eth2NearRelay {
         let end_period = BeaconRPCClient::get_period_for_slot(last_finalized_slot_on_eth);
         info!(target: "relay", "Last finalized slot/period on ethereum={}/{}", last_finalized_slot_on_eth, end_period);
 
-        if last_finalized_slot_on_eth - last_finalized_slot_on_near > 500 {
+        if last_finalized_slot_on_eth - last_finalized_slot_on_near >= self.max_blocks_for_finalization {
             info!(target: "relay", "Too big gap between slot of finalized block on Near and Eth. Sending hand made light client update");
             self.send_hand_made_light_client_update(last_finalized_slot_on_near);
```

Comment:
Please display the result of sending updates & submit transactions as info! because it is not clear when monitoring the output.
Also it should contain the link of the transaction `${url}/transactions/${txId}`, where the URL `https://explorer.${this.nearNetworkId}.near.org`

Reply:
fixed https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/e5ac1ac3bef3c9ecab2a80a5d67ea3fc8b1f73f4